Makefile

```diff
@@ -5,8 +5,8 @@ help:
 	@echo 'Makefile for YABT                                                  '
 	@echo '                                                                   '
 	@echo 'Usage:                                                             '
-	@echo '   make test       Run full test suite with active Python and PEP8 '
-	@echo '   make quicktest  Run test suite with active Python and PEP8      '
+	@echo '   make test       Run full test suite with active Python and FLAKE8 '
```

Comment:
oh no, the indentation on the closing `'` is off now! 😱

Reply:
the evil find & replace!